routes/web.php

```diff
@@ -181,7 +181,7 @@
         Route::get('create', [VaultController::class, 'create'])->name('vault.create');
         Route::post('', [VaultController::class, 'store'])->name('vault.store');
 
-        Route::middleware(['vault'])->prefix('{vault}')->group(function () {
+        Route::middleware('can:any,\App\Models\Vault,vault')->prefix('{vault}')->group(function () {
```

Comment:
Does it have to be the full path to the model here?

Reply:
Also, are spaces missing here?

Reply:
> Does it have to be the full path to the model here?

It's mandatory for us because all methods are not using auto-binding. If we were using auto-binding everywhere, then we would be able to write the middleware like: `can:any,vault`